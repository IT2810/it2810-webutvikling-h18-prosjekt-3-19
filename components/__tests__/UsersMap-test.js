import "react-native";
import React from "react";
import UsersMap from "../UsersMap";
import renderer from "react-test-renderer";

jest.mock("react-native-maps", () => "MapView");
it("renders correctly", () => {
  const tree = renderer
    .create(<UsersMap userLocation={null} usersPlaces={[]} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
