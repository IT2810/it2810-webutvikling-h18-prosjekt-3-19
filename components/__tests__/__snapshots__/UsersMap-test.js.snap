// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "height": 550,
      "width": "100%",
    }
  }
>
  <MapView
    initialRegion={
      Object {
        "latitude": 63.430515,
        "latitudeDelta": 0.0622,
        "longitude": 10.395053,
        "longitudeDelta": 0.0421,
      }
    }
    region={null}
    style={
      Object {
        "height": "100%",
        "width": "100%",
      }
    }
  />
</View>
`;
